pkgs/development/cuda-modules/gpus.nix

```diff
@@ -193,33 +196,35 @@
     dontDefaultAfter = null;
     maxCudaVersion = null;
   }
-  {
-    # NVIDIA B100 Accelerated
-    archName = "Blackwell";
-    computeCapability = "10.0a";
-    isJetson = false;
-    minCudaVersion = "12.8";
-    dontDefaultAfter = null;
-    maxCudaVersion = null;
-  }
-  {
-    # NVIDIA Blackwell
-    archName = "Blackwell";
-    computeCapability = "10.1";
-    isJetson = false;
-    minCudaVersion = "12.8";
-    dontDefaultAfter = null;
-    maxCudaVersion = null;
-  }
-  {
-    # NVIDIA Blackwell Accelerated
-    archName = "Blackwell";
-    computeCapability = "10.1a";
-    isJetson = false;
-    minCudaVersion = "12.8";
-    dontDefaultAfter = null;
-    maxCudaVersion = null;
-  }
+  /*
+    {
+      # NVIDIA B100 Accelerated
+      archName = "Blackwell";
+      computeCapability = "10.0a";
+      isJetson = false;
+      minCudaVersion = "12.8";
+      dontDefaultAfter = null;
+      maxCudaVersion = null;
```

Comment:
Instead of removing metadata let's add some boolean for filtering it out in default capabilities computation.

Reply:
So you are suggesting setting `dontDefaultAfter = "12.8"` with a comment?